lib/disk/modules/QcowDisk.rb

```diff
@@ -252,6 +284,16 @@ def read_entries(n)
     entries
   end
 
+  def refcount_order
+    # https://github.com/qemu/qemu/blob/v2.2.0/docs/specs/qcow2.txt#L108
+    version < 3 ? 4 : @header['refcount_order']
+  end
+
+  def header_length
+    # https://github.com/qemu/qemu/blob/v2.2.0/docs/specs/qcow2.txt#L115
+    version < 3 ? 72 : @header['header_length']
```

Comment:
If SIZEOF_QCOW_HEADER_V2 is 72, can you please replace the magic number here with the constant reference?